components/layout-switcher.tsx

```
@@ -95,146 +101,175 @@ export function LayoutSwitcher({ inline = false }: LayoutSwitcherProps) {
+                      <p
+                        className={`text-xs transition-colors duration-300 ${textClasses.desc}`}
+                      >
+                        {description}
+                      </p>
+                    </div>
                   </div>
+                  {isActive && (
+                    <div
+                      className={`flex items-center gap-2 px-3 py-1 rounded-full text-xs font-semibold shadow-lg ${getBadgeClasses(color)}`}
+                    >
+                      <div className="w-2 h-2 bg-white rounded-full animate-pulse"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)